codebase/superdarn/src.lib/tk/cnvmodel.1.0/src/cnvmodel.c

```diff
@@ -556,20 +563,20 @@ struct model *interp_coeffs(int ih, float tilt, float mag, float cang, int imod)
                 cimag(Dn[k])*afn*mgn*dtp + cimag(En[k])*afp*mgp*dtn +
                 cimag(Fn[k])*afn*mgp*dtn + cimag(Gn[k])*afp*mgn*dtn +
                 cimag(Hn[k])*afn*mgn*dtn);
-      } else {
-        k = l*(ptr->ltop+1)+m;
-
-        ptr->aoeff_p[k] = CMPLX(creal(Ap[k])*afp*mgp*dtp +
-                creal(Bp[k])*afn*mgp*dtp + creal(Cp[k])*afp*mgn*dtp +
-                creal(Dp[k])*afn*mgn*dtp + creal(Ep[k])*afp*mgp*dtn +
-                creal(Fp[k])*afn*mgp*dtn + creal(Gp[k])*afp*mgn*dtn +
-                creal(Hp[k])*afn*mgn*dtn, cimag(Ap[k])*afp*mgp*dtp +
-                cimag(Bp[k])*afn*mgp*dtp + cimag(Cp[k])*afp*mgn*dtp +
-                cimag(Dp[k])*afn*mgn*dtp + cimag(Ep[k])*afp*mgp*dtn +
-                cimag(Fp[k])*afn*mgp*dtn + cimag(Gp[k])*afp*mgn*dtn +
-                cimag(Hp[k])*afn*mgn*dtn);
-
-      }
+      // TODO: I think this is duplicate code which makes more susceptable bugs
```

Comment:
This is not duplicate code and should not be removed - `ptr->aoeff_n` and `ptr->aoeff_p` are both necessary.